### ISSUE TITLE
Hyper-params settings for MNLI fine-tuning using Albert-v2

### ISSUE THREAD

**Issue by agupta74** [google-research/ALBERT#123]
What hyper-parameter settings (learning rate, batch size etc.) have been used for fine-tuning Albert-v2 module on MNLI task? I am seeing accuracy of ~82.6 as compared to 84.6 reported in this repo for Albert v2.

**Comment by MichaelZhouwang** [comment]
I've followed the hyper parameters presented in the paper but only got accuracy of ~83.0. Have you figured out how to get the reported result? Thanks!

**Comment by kavin525zhang** [comment]
hi，I run the code according to the official command, but it doesn't reproduce the results,  only get accuracy of ~52.6, can you tell me where is wrong?
